docs/docs/specification.md

```diff
@@ -42,13 +44,54 @@ func (n *Node) Hash() []byte {
 **Public Key**: Public keys of drand nodes are points on the G1 group of the
 BLS12-381 curve. See the [curve](#drand-curve) section for more information.
 
+### Drand network
+Each group of nodes that runs a specific process (e.g., one with the same set of parameters) constitutes a separate network. 
+Each network will act as an independent beacon generator. 
+
 ### Drand beacon
 
 A drand beacon is what the drand network periodically creates and that can be
 used to derive the randomness. A beacon contains the signature of the previous
 beacon generated, the round of this beacon and signature. See the [beacon
 chain](#beacon-chain) section for more information.
 
+### Beacon ID
+The `Beacon ID` is the unique identifier among existing beacon processes running on a drand node. Thanks to
```

Comment:
We should explain here the subtle difference with chainhash and when one is used vs the other.
BeaconID is manually set by operators and is just a regular identifier while ChainHash binds the public key of the network and is therefore a cryptographic identifier and is needed by clients. Clients don't really deal with beacon IDs at all (or do they ? They shouldn't really right?)

Reply:
Done!